### PR TITLE
Header, Footer, Main, and Current Price

### PR DESCRIPTION
**Pull request by fhzno**

### CODE REVIEW

**Comment by wytman** [comment]
Good start.

File: src/pages/landing.tsx [hunk 1]
```diff
@@ -1,6 +1,39 @@
...
-import { Component } from '../components';
+import { Header, Footer, Main } from '../components';
+import { apiUrl } from '../utils/constants';
 
-const Landing: React.FC = () => <Component />;
+interface IState {
+  current_price: string;
+}
 
+class Landing extends React.Component<{}, IState> {
```
Comment (kali-maha) [inline src/pages/landing.tsx:9]: What would you say if we went with Functional Components? They are more light than classes and require less code

File: src/pages/landing.tsx [hunk 2]
```diff
@@ -1,6 +1,39 @@
...
+
+  componentDidMount() {
+    this.getCurrentPrice();
+  }
+
+  getCurrentPrice() {
+    fetch(apiUrl)
+      .then(response => response.json())
+      .then(data => this.setState({ current_price: data.tickers[0].converted_last['usd'] }));
+  }
```
Comment (kali-maha) [inline src/pages/landing.tsx:25]: We could also do this `await`

**Review by kali-maha — COMMENTED** [review]
Thanks buddy, let some comments

File: src/pages/landing.tsx (hunk 2 above)
Comment (fhzno) [reply src/pages/landing.tsx:25]: Good point, I actually meant to do this Axios as well so will swap that out for an async call

**Review by kali-maha — APPROVED** [review]

**Pull request merged**